plugins/modules/s3_lifecycle.py

```diff
@@ -23,16 +23,28 @@
       - Name of the S3 bucket.
     required: true
     type: str
+  abort_incomplete_multipart_upload_days:
+    description:
+      - Specifies the days since the initiation of an incomplete multipart upload that Amazon S3 will wait before permanently removing all parts of the upload.
+    type: int
   expiration_date:
     description:
       - Indicates the lifetime of the objects that are subject to the rule by the date they will expire.
       - The value must be ISO-8601 format, the time must be midnight and a GMT timezone must be specified.
+      - This cannot be specified with C(expire_object_delete_marker)
```

Comment:
```suggestion
      - This cannot be specified with I(expire_object_delete_marker)
```

Reply:
What does `I()` mean? Is this markdown or something else?